pkg/cmd/reset.go

```diff
@@ -90,5 +99,23 @@ func runReset(opts *resetOpts) error {
 		return errors.Wrap(err, "failed to initialize State")
 	}
 
+	fmt.Println("The following nodes will be resetted: ")
+	for _, node := range s.Cluster.ControlPlane.Hosts {
+		fmt.Printf("\t+ reset control plane node %q (%s)\n", node.Hostname, node.PrivateAddress)
+	}
+	for _, node := range s.Cluster.StaticWorkers.Hosts {
+		fmt.Printf("\t+ reset static worker nodes %q (%s)\n", node.Hostname, node.PrivateAddress)
+	}
+
+	confirm, err := confirmApply(opts.AutoApprove)
```

Comment:
Nit: since we're now using this function for other commands as well, we should probably rename it to something like `confirmOperation` and move it to a more generic place (e.g. to the [`shared.go` file](https://github.com/kubermatic/kubeone/blob/master/pkg/cmd/shared.go))

Reply:
Makes sense! I absolutely agree with you there :)